transformation-scripts/semopenalex-domains.py

```diff
@@ -0,0 +1,306 @@
+                        domain_description = clean(domain_description)
+                        domain_graph.add((domain_uri, SKOS.note, Literal(domain_description, datatype=XSD.string)))
+
+                    # siblings
+                    domain_siblings = json_data['siblings']
+                    if not domain_siblings is None:
+                        for sibling in domain_siblings:
+                            sibling_id = sibling['id'].replace("https://openalex.org/domains/", "")
+                            sibling_uri = URIRef(soa_namespace_domains+sibling_id)
+                            domain_graph.add((domain_uri, SKOS.related, sibling_uri))
+
+                    # ids (wikidata, wikipedia)
+                    domain_wikidata = json_data.get('ids').get('wikidata')
+                    if not domain_wikidata is None:
+                        domain_wikidata = clean_url(domain_wikidata)
+                        domain_graph.add((domain_uri,OWL.sameAs, URIRef(domain_wikidata)))
+
+                    domain_wikipedia = json_data.get('ids').get('wikipedia')
+                    if not domain_wikipedia is None:
+                        domain_wikipedia = clean_url(domain_wikipedia)
+                        domain_graph.add((domain_uri,RDFS.seeAlso, Literal(domain_wikipedia, datatype=XSD.anyURI)))
+
+                    # works_count
+                    domain_works_count = json_data['works_count']
+                    if not domain_works_count is None:
+                        domain_graph.add((domain_uri, works_count_predicate, Literal(domain_works_count, datatype=XSD.integer)))
+
+                    # cited_by_count
+                    domain_cited_by_count = json_data['cited_by_count']
+                    if not domain_cited_by_count is None:
+                        domain_graph.add((domain_uri, cited_by_count_predicate, Literal(domain_cited_by_count, datatype=XSD.integer)))
+
+                    # created_date
+                    domain_created_date = json_data['created_date']
+                    if not domain_created_date is None:
+                        domain_graph.add((domain_uri, DCTERMS.created, Literal(domain_created_date, datatype=XSD.date)))
+
+                    # updated_date
+                    domain_updated_date = json_data['updated_date']
+                    if not domain_updated_date is None:
+                        domain_updated_date = clean_date(domain_updated_date)
+                        domain_graph.add((domain_uri, DCTERMS.modified, Literal(domain_updated_date, datatype=XSD.date)))
+
+
+                    i += 1
+                    print('Processed domains entity {} lines'.format(i))
+
+                except Exception as e:
+                    print(str((e)) + ' Error in domains entity line ' + str(i + 1 + error_count))
+                    error_count += 1
+                    pass
+
+    # Write the graph to the .trig file
+    g.write(domain_graph.serialize(format='trig'))
+
```

Comment:
missing `domain_graph = Graph(identifier=context)`

Reply:
Not needed here because the entire rdf-graph data is wrote to the file once at the end of the script. Then no new graph is needed.